app/src/main/java/com/rob729/quiethours/util/EndAlarm.kt

```diff
@@ -24,8 +24,15 @@ class EndAlarm(appContext: Context, workerParams: WorkerParameters) :
 
         val audioManager =
             applicationContext.getSystemService(Context.AUDIO_SERVICE) as AudioManager
-        audioManager.ringerMode = AudioManager.RINGER_MODE_NORMAL
         StoreSession.writeInt(AppConstants.BEGIN_STATUS, StoreSession.readInt(AppConstants.BEGIN_STATUS) - 1)
+        if (StoreSession.readInt(AppConstants.BEGIN_STATUS)> 0) {
```

Comment:
Change it to audioManager.ringerMode = if...

Reply:
Done